test/catalog-fixtures/DefaultCatalogFixture.json

```diff
@@ -30,6 +30,21 @@
 			"paths": [
 				"https://eslint.org/docs/rules/no-inner-declarations"
 			]
+		},
```

Comment:
Adding typescript rules here to include more coverage in existing end to end tests.